packages/backend/src/open_payments/payment_pointer/model.ts

```diff
@@ -48,6 +48,11 @@ export class PaymentPointer
   // select sum(`withdrawalAmount`) from `webhookEvents` where `withdrawalAccountId` = `paymentPointer.id`
   public totalEventsAmount!: bigint
   public processAt!: Date | null
+  public deactivatesAt!: Date | null
+
+  public get isActive() {
+    return !this.deactivatesAt || new Date(this.deactivatesAt) > new Date()
```

Comment:
I'm understanding we need to wrap `new Date(this.deactivatesAt)` in `new Date()` since `objection` doesn't properly cast the value into a javascript date when retrieving from the DB?

Reply:
Unwrapped may work, not sure if I tried it. I will test that use it if it does.

Reply:
yeah I don't need to construct a date for deactivates, as it's already a date. good catch